### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by monsieurp**
Here's a few improvements for this eselect module. Please have a look and let me know. :)

### CODE REVIEW

**Comment by monsieurp** [comment]
ping @elprans :'(

**Review by elprans — COMMENTED** [review]
Hm, this doesn't seem to work for me:
```shell
$ eselect electron list
Available Electron Slots
!!! Warning: No slots available.
```

File: electron.eselect [hunk 1]
```diff
@@ -46,20 +46,28 @@ lib_dir() {
...
+		if [[ -x "${tmp}" ]]; then
+			found_slots+=( "${slot}" )
+		fi
+	done < \
+		<(find "${USR_PATH}/$(lib_dir)/" \
+		-mindepth 1 \
+		-maxdepth 1 \
+		-type d \
+		-name 'electron-*' \
+		-print0 | \
```
Comment (elprans) [inline electron.eselect:65]: This is the culprit, `-print0` interacts badly with sed.  Why use it?

**Comment by monsieurp** [comment]
@elprans I completely forgot about this PR. Do you want me to carry on with it or can I close it?

**Pull request closed without merging**